src/large_pages/node_large_page.cc

```diff
@@ -282,6 +282,25 @@ bool IsSuperPagesEnabled() {
 }
 #endif
 
+class MmapPointer {
+ public:
+  void
+  DoMmap(void* start, size_t size_, int prot, int flags, int fd, int offset) {
+    mem = mmap(start, size_, prot, flags, fd, offset);
+    size = size_;
+  }
+  bool operator==(void* rhs) { return mem == rhs; }
+  operator void*() { return mem; }
+  void operator= (void* rhs) { mem = rhs; }
+  ~MmapPointer() {
+    if (mem != nullptr && mem != MAP_FAILED && munmap(mem, size) != 0)
+      PrintSystemError(errno);
+  }
+ private:
```

Comment:
```suggestion

 private:
```

(The linter should complain about this, I think)